palladian-commons/src/test/java/ws/palladian/helper/StopWatchTest.java

```diff
@@ -0,0 +1,192 @@
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import ws.palladian.helper.date.DateHelper;
+
+import java.lang.reflect.Method;
+
+@RunWith(PowerMockRunner.class)
+public class StopWatchTest {
+
+  @Rule public final ExpectedException thrown = ExpectedException.none();
+  @Rule public final Timeout globalTimeout = new Timeout(10000);
+
+  @PrepareForTest({System.class, StopWatch.class})
+  @Test
+  public void testGetStartTime() {
+
+    PowerMockito.mockStatic(System.class);
+
+    final StopWatch objectUnderTest = new StopWatch();
+    PowerMockito.when(System.currentTimeMillis())
+            .thenReturn(0L);
+
+    Assert.assertEquals(0L, objectUnderTest.getStartTime());
+  }
+
+  @PrepareForTest({System.class, StopWatch.class})
+  @Test
+  public void testGetTotalElapsedTime() {
+
+    PowerMockito.mockStatic(System.class);
+
+    final StopWatch objectUnderTest = new StopWatch();
+    objectUnderTest.setCountDown(0L);
+    PowerMockito.when(System.currentTimeMillis())
+            .thenReturn(0L)
+            .thenReturn(6_231L)
+            .thenReturn(95_648L);
+
+    Assert.assertEquals(0L, objectUnderTest.getTotalElapsedTime());
+    Assert.assertEquals(6_231L, objectUnderTest.getTotalElapsedTime());
+    Assert.assertEquals(95_648L, objectUnderTest.getTotalElapsedTime());
+  }
+
+  @Test
+  public void testGetCountDown() {
+
+    final StopWatch objectUnderTest = new StopWatch();
+
+    objectUnderTest.setCountDown(0L);
+    Assert.assertEquals(0L, objectUnderTest.getCountDown());
+
+    objectUnderTest.setCountDown(5_432L);
+    Assert.assertEquals(5_432L, objectUnderTest.getCountDown());
+  }
+
+  @PrepareForTest({System.class, StopWatch.class})
+  @Test
+  public void testGetElapsedTime() {
+
+    PowerMockito.mockStatic(System.class);
+
+    final StopWatch objectUnderTest = new StopWatch();
+    PowerMockito.when(System.currentTimeMillis())
+            .thenReturn(0L)
+            .thenReturn(0L)
+            .thenReturn(0L)
+            .thenReturn(4_758L);
+
+    objectUnderTest.start();
+    objectUnderTest.stop();
+
+    Assert.assertEquals(0L, objectUnderTest.getElapsedTime(false));
+    Assert.assertEquals(0L, objectUnderTest.getElapsedTime(true));
+
+    objectUnderTest.start();
+    objectUnderTest.stop();
+
+    Assert.assertEquals(4_758L, objectUnderTest.getElapsedTime(false));
+    Assert.assertEquals(4L, objectUnderTest.getElapsedTime(true));
+  }
+
+  @PrepareForTest({DateHelper.class, StopWatch.class, System.class})
+  @Test
+  public void testGetElapsedTimeStringAndIncrement()
+          throws Exception {
+
+    PowerMockito.mockStatic(System.class);
+    PowerMockito.mockStatic(DateHelper.class);
+
+    final StopWatch objectUnderTest = new StopWatch();
+    final Method formatDurationMethod =
+            DTUMemberMatcher.method(DateHelper.class, "formatDuration", long.class, long.class);
```

Comment:
It is possible we might not need to use the Member Matcher from DeeptestUtils, which is one less dependency for them to add (Powermock has its own one, but can be buggy, which is why we wrote our own). Speak to me about how to use powermock's one.